README.MD

```diff
@@ -4,6 +4,18 @@ Subject - CSTD
 
 In this educational project was implemented a recognition system of European-style car license plates.
 
+## How to run:
+Install requirements:
```

Comment:
I can not understand: you added info ablout AI, but how you implemented it?

Reply:
Where is code?

Reply:
Actually, there are deep learning (AI) mechanics, algorithms and even recognition models that are already implemented in pre-compiled EasyOCR package I used in my project. Bunch of them are based on Pytorch, machine learning framework. Also, haar-cascade I used in OpenCV part of project is actualy a machine learning-based approach where a lot of positive and negative images are used to train the classifier. I just took the complete one. So basically it could be represented as already trained AI model. All references and acknowledgements I placed in README file.